### PR TITLE
Replace lodash with native methods

### PR DESCRIPTION
**Pull request by XhmikosR**
Draft until the other PRs are sorted and also because I had to change the TS target.

I'll check if there's any other way without changing the target.

TODO:

- [ ] Make sure everything works because the native methods aren't 1-to-1 replacement when it comes to null etc

EDIT: So it seems there are some workarounds with `downlevelIteration` etc, but I'm not sure it's worth it. Personally I think we should change the target to be in sync with the `engines` version and/or make it a major version bump.

Not super familiar with TS, so I'd leave this to you @tancredi :)

### CODE REVIEW

**Comment by tancredi** [comment]
heyhey @XhmikosR - this is waay overkill to get rid of `uniq` and `startsWith` - I'm happy to re-implement this methods in utils, it really ain't gonna be much work - question: why is this PR also making changes to `files` in `package.json`?

Thanks for the contribution again :)

**Comment by XhmikosR** [comment]
https://github.com/tancredi/fantasticon/pull/93#discussion_r553808584

I think we all agree that less dependencies is always better :) That being said, I don't think we need a new util. The vanilla function should suffice, but if not we could either adapt any specific parts than adding a new util, but that's your call. I mean, I don't even know if things can break here; it could be fine as is :)

**Review by tancredi — APPROVED** [review]
Big fan of this 👍

File: tsconfig.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "outDir": "lib",
     "rootDir": "src",
     "sourceMap": true,
+    "target": "ES6",
```
Comment (XhmikosR) [inline tsconfig.json:11]: @tancredi you might want to experiment with setting this to ES2017 or newer since you already support only Node.js >= 10.

Comment (tancredi) [reply tsconfig.json:11]: Sure, but one thing at a time ;) this solves a problem and is good for now

Comment (XhmikosR) [reply tsconfig.json:11]: I know, just wanted to note this so that you have a look at later

**Pull request merged**